src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1095,6 +1097,18 @@ namespace winrt::TerminalApp::implementation
         return std::nullopt;
     }
 
+
+    // Method Description:
+    // - returns a com_ptr to the currently focused tab. This might return null,
+    //   so make sure to check the result!
+    winrt::com_ptr<Tab> TerminalPage::_GetFocusedTab()
```

Comment:
Heads up - this'll probably merge conflict, since I added the same function in #6567 😋

Reply:
It didn't conflict, maybe because I copied it straight from #6567 into the exact place that you had put it.